staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -274,6 +274,7 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 		}, context.StopCh)
 	})
 
+	s.GenericAPIServer.RegisterDestroyFunc(crdHandler.destroy)
```

Comment:
can we move this up to right after we initialize crdHandler?

Reply:
done